pkg/analysis/passes/screenshots/screenshots.go

```diff
@@ -39,24 +49,59 @@ func checkScreenshotsExist(pass *analysis.Pass) (interface{}, error) {
 		explanation := "Screenshots are displayed in the Plugin catalog. Please add at least one screenshot to your plugin.json."
 		pass.ReportResult(pass.AnalyzerName, screenshots, "plugin.json: should include screenshots for the Plugin catalog", explanation)
 		return data.Info.Screenshots, nil
-	} else {
-		reportCount := 0
-		for _, screenshot := range data.Info.Screenshots {
-			if strings.TrimSpace(screenshot.Path) == "" {
-				reportCount++
-				pass.ReportResult(pass.AnalyzerName, screenshots, fmt.Sprintf("plugin.json: invalid empty screenshot path: %q", screenshot.Name), "The screenshot path must not be empty.")
-			}
-		}
+	}
 
-		if reportCount > 0 {
-			return nil, nil
+	reportCount := 0
+	for _, screenshot := range data.Info.Screenshots {
+		if strings.TrimSpace(screenshot.Path) == "" {
+			reportCount++
+			pass.ReportResult(pass.AnalyzerName, screenshots, fmt.Sprintf("plugin.json: invalid empty screenshot path: %q", screenshot.Name), "The screenshot path must not be empty.")
+		} else if !validImageType(filepath.Join(archiveDir, screenshot.Path)) {
+			reportCount++
+			pass.ReportResult(pass.AnalyzerName, screenshotsType, fmt.Sprintf("invalid screenshot image type: %q. Accepted image types: %q", screenshot.Path, acceptedImageTypes), "The screenshot image type invalid.")
```

Comment:
This error can be confusing for users if for example they have a typo in their plugin json or the wrong path. We'll be telling them the image type is wrong but in reality the image path is wrong.

Let's have an error if the image path can't be found and a separate one for the image type.